www/templates/default/html/manager/Controller.tpl.php

```diff
@@ -40,6 +41,7 @@
 $page->addStyleDeclaration("#dcf-mobile-toggle-menu {display: none!important}");
 
 //javascript
+$page->addScriptDeclaration('WDN.setPluginParam("idm", "logout", "' .  Util::getBaseURL() . '/manager/logout");');
```

Comment:
Somehow for this we're getting "https://events-dev.unl.edu/manager/https/manager/logout"